src/interfaces/api-filing-interface.ts

```diff
@@ -53,7 +53,8 @@ export interface ApiFilingIF {
     // COD filings only
     changeOfDirectors?: any // some object
 
-    changeOfName?: ChangeOfNameIF
```

Comment:
I did a global search and it looks like this wasn't used anywhere, so probably a left-over from a previous design.